src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -398,6 +400,26 @@ public void Timeout_TooShort_AllPendingOperationsCanceled()
             }
         }
 
+        [Fact]
+        public async Task SendAsync_UserAgent_CorrectlyWritten()
+        {
+            string userAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.18 Safari/537.36";
+
+            await LoopbackServer.CreateClientAndServerAsync(async uri =>
+            {
+                using (var client = CreateHttpClient())
+                {
+                    var message = new HttpRequestMessage(HttpMethod.Post, uri);
```

Comment:
nit: Why is this using HttpMethod.Post?  There is no request content being posted.  This should be using HttpMethod.Get.  The loopback server here probably doesn't care but it seems odd to write the test that way.